refnx/reflect/test/simulation.lgt

```diff
@@ -0,0 +1,57 @@
+C1 26.659000000000002 0.0
+C2 26.659000000000002 0.0
+C3 26.659000000000002 0.0
+N4 9.36 0.0
+C5 19.998 0.0
+C6 19.998 0.0
+O7 5.803000000000001 0.0
+P8 5.13 0.0
+O9 5.803000000000001 0.0
+O10 5.803000000000001 0.0
+O11 5.803000000000001 0.0
+C12 19.998 0.0
+C13 13.317 0.0
+O14 5.803000000000001 0.0
+C15 6.646000000000001 0.0
+O16 5.803000000000001 0.0
+C17 19.998 0.0
+C18 19.998 0.0
+C19 19.998 0.0
+C20 19.998 0.0
+C21 19.998 0.0
+C22 19.998 0.0
+C23 19.998 0.0
+C24 19.998 0.0
+C25 19.998 0.0
+C26 19.998 0.0
+C27 19.998 0.0
+C28 19.998 0.0
+C29 19.998 0.0
+C30 19.998 0.0
+C31 19.998 0.0
+C32 19.998 0.0
+C33 26.659000000000002 0.0
+C34 19.998 0.0
+O35 5.803000000000001 0.0
+C36 6.646000000000001 0.0
+O37 5.803000000000001 0.0
+C38 19.998 0.0
+C39 19.998 0.0
+C40 19.998 0.0
+C41 19.998 0.0
+C42 19.998 0.0
+C43 19.998 0.0
+C44 19.998 0.0
+C45 19.998 0.0
+C46 19.998 0.0
+C47 19.998 0.0
+C48 19.998 0.0
+C49 19.998 0.0
+C50 19.998 0.0
+C51 19.998 0.0
+C52 19.998 0.0
+C53 19.998 0.0
+C54 26.659000000000002 0.0
+OW 0.0 0.0
```

Comment:
water has zero scattering length... Is this meant to enable contrast variation somehow?

Reply:
See point 2